Common/ImageSamplers/itkImageGridSampler.hxx

```diff
@@ -258,7 +258,7 @@ ImageGridSampler<TInputImage>::SingleThreadedGenerateData(const TInputImage &
             inputImage.TransformIndexToPhysicalPoint(index, tempSample.m_ImageCoordinates);
 
             // Equivalent to `mask->IsInsideInWorldSpace(tempSample.m_ImageCoordinates)`, but much faster.
-            if (mask->IsInsideInObjectSpace(
+            if (mask->MaskType::IsInsideInObjectSpace(
                   worldToObjectTransform.WorldToObjectTransformType::TransformPoint(tempSample.m_ImageCoordinates)))
```

Comment:
@mstaring @stefanklein  Can we possibly still optimize for if the mask and the input image have the same geometry?
https://github.com/SuperElastix/elastix/blob/217a07cc455b2afad55a260584c688e50672466d/Common/itkComputeImageExtremaFilter.hxx#L138

Reply:
Reported as issue at:
- https://github.com/SuperElastix/elastix/issues/1023